src/runtime/transformers/shiki/highlighter.ts

```diff
@@ -148,12 +148,12 @@ export const useShikiHighlighter = createSingleton((opts?: Exclude<ModuleOptions
 
     return lines.map((line, lineIndex) => ({
       type: 'element',
-      tag: 'div',
+      tag: 'span',
       props: {
         class: ['line', highlights.includes(lineIndex + 1) ? 'highlight' : ''].join(' ').trim(),
         line: lineIndex + 1
       },
-      children: line.map(tokenSpan)
+      children: [...line.map(tokenSpan), { type: 'text', value: '\n' }]
```

Comment:
```suggestion
      children: [...line.map(tokenSpan), { type: 'text', value: lineIndex !== lines.length - 1 ? '\n' : '' }]
```
We could also add a check if it is the last line, even though the browser ignores the trailing \n in the last line

Reply:
I like it, let's prevent adding `\n` to last line.

Also, WDYT about adding `\n` directly into `line[line.length - 1].content`. This will prevent adding one extra tag to HTML

Reply:
LGTM